specification/databoxedge/resource-manager/Microsoft.DataBoxEdge/preview/2020-07-01-preview/databoxedge.json

```diff
@@ -136,6 +136,52 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.DataBoxEdge/skus": {
+      "get": {
+        "tags": [
+          "Skus"
+        ],
+        "summary": "List all the available Skus and information related to them.",
+        "operationId": "Skus_List",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "name": "$expand",
+            "in": "query",
+            "description": "Specify $filter='location eq <location>' to filter on location resource or Specify $skipToken=<token> to populate the next page in the list.",
+            "required": false,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Lists the available Skus and information related to them.",
+            "schema": {
+              "$ref": "#/definitions/DataBoxEdgeSkuList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
```

Comment:
`DataBoxEdgeSkuList` has `nextLink` property. Is it correct here as `null`?

Reply:
This API is not used. But changed it.